finch/src/services/email.js

```diff
@@ -0,0 +1,51 @@
+import nodemailer from 'nodemailer';
+
+import config from '../../config/index.js';
+
+const { emailSender, smtp } = config;
+
+export default class Email {
+
+  constructor() {
+    this.sender = emailSender;
+  }
+
+  init() {
+    this.transporter = nodemailer.createTransport({
```

Comment:
you my want to check here if `this.transporter` exists already.

Reply:
I guess the need for checking if `this.transporter` exist or not will be removed if we include this initialization in constructor only